ext/encryptors/nop/nop_encrypt.c

```diff
@@ -130,6 +130,87 @@ nop_sizing(WT_ENCRYPTOR *encryptor, WT_SESSION *session, size_t *expansion_const
 }
 /*! [WT_ENCRYPTOR sizing] */
 
+/*! [WT_ENCRYPTOR customize] */
+/*
+ * nop_customize -- The customize function creates a customized encryptor.
```

Comment:
This line is causing the PR testing to fail. The `--` needs to end its line and the description needs to be on the next line, indented 4 spaces. So unjoin `-- The`.

Reply:
You should be able to run the `s_all` script outside of our PR testing. It is in `dist/s_all`.

Reply:
That'll bring back the hanging-indent comments. I guess maybe the way to deal with this is to move the rest of the comment inside the function.

(I've been running s_all, but I only ran s_clang-format before pushing that change. Bad idea, I guess.)

Reply:
I checked the clang-format docs and there doesn't seem to be any way to control how it formats comments, and putting two separate comments in a row before the function also fails function.py, so I what I just pushed is probably the best choice.